bliss/models/decoder.py

```diff
@@ -349,7 +355,7 @@ def sample_prior(self, batch_size=1):
     @staticmethod
     def _get_log_fluxes(fluxes):
         log_fluxes = torch.where(
-            fluxes > 0, fluxes, torch.ones(*fluxes.shape).to(device)
+            fluxes > 0, fluxes, torch.ones(*fluxes.shape).to(fluxes.device)
```

Comment:
Would it work to replace `torch.ones(*fluxes.shape).to(fluxes.device)` with `torch.ones_like(fluxes)`?

Reply:
Oh right, this is much cleaner. I changed this in the next commit.